Userland/Libraries/LibGUI/Application.h

```diff
@@ -75,6 +75,7 @@ class Application : public Core::Object {
     }
     void notify_drag_cancelled(Badge<WindowServerConnection>);
 
+    RefPtr<Window> create_dummy_child_window(i32 window_server_client_id, i32 window_parent_id);
```

Comment:
I don't understand the purpose of this API. This sounds like something that belongs in a test program, not as a public Application API.

Reply:
I've moved this into a private method for the server. Also went on a rampage and removed all the changes from `Application` as they weren't technically necessary.